shell/probe/install.ps1

```diff
@@ -74,7 +82,8 @@ if(Test-Path -path $probePath -PathType Leaf) {
 LogMessage "Determining OS"
 $dos = "windows-" + $Env:PROCESSOR_ARCHITECTURE
 $token=RegisterEndpoint
-[System.Environment]::SetEnvironmentVariable("PRELUDE_TOKEN", $token, "User")
+[Environment]::SetEnvironmentVariable("PRELUDE_TOKEN", $token, "User")
+[Environment]::SetEnvironmentVariable("PRELUDE_API", $PRELUDE_API, "Process")
```

Comment:
the ScheduledTask will start a different process I think, so you need to add this line to the scheduled task action I believe